### DISCUSSION TITLE
DAI 2023-2024 - Pratical work 1 - Slimani Walid (SlWa99)

### DISCUSSION THREAD

**Discussion by SlWa99** [heig-vd-dai-course/heig-vd-dai-course#88] [Show and tell]
https://github.com/SlWa99/WSI_DAI_Pratical_Work_1

**Comment by ludelafo** [comment]
Great! Could you tell us a bit more about the idea you have for the CLI tool?

It is OK if you want to replicate the demo made by Hadrien in class if you want. If you do so, we will ask you to add a few more commands as the CLI presented by Hadrien is a fairly simple one.

**Comment by SlWa99** [comment]
My CLI application can execute 2 commands. The first is called Replace. It replaces a given word in a text with another word. It's possible to replace the first occurrence of the old word, or all occurrences. The second command is called convert. It permits to convert the input image to black and white. The output image format can be jpeg (or jpg) or png.
